app/controllers/home_controller.rb

```diff
@@ -1,3 +1,11 @@
 class WeekFourAssessmentApp
-  # Your code will go here.
+  get '/' do
+    @name = 'Francesco'
+    erb :welcome
+  end
+
+  get '/name/:name' do
+    "<p>Hello #{params[:name]}</p>"
```

Comment:
This is NOT what we are looking for. We want you to create a user and pass on that info to the view template.

Reply:
I'll change that